hbase-server/src/test/java/org/apache/hadoop/hbase/util/ProcessBasedLocalHBaseCluster.java

```diff
@@ -187,11 +187,19 @@ public void startHBase() throws IOException {
   }
 
   public void startRegionServer(int port) {
-    startServer(ServerType.RS, port);
+    try {
+      startServer(ServerType.RS, port);
+    } catch (IOException e) {
+      throw new RuntimeException(e);
```

Comment:
This changes behavior? What you thinking here?

Reply:
It just because the new impl for getting the port throws IOException - it has to be caught and dealt with somewhere - until cleanup, it was here. I put this up early since I saw the other issue.